invenio_drafts_resources/records/api.py

```diff
@@ -227,3 +228,16 @@ def edit(cls, record):
                 fork_version_id=record.revision_id,
             )
         return draft
+
+    @classmethod
+    def cleanup_drafts(cls, td):
+        """Clean all the soft deleted drafts.
+
+        The drafts in the last timedelta span of time won't be soft deleted.
+        """
```

Comment:
minor: the wording is confusing... What does "clean up" means if after it says "won't be soft deleted"... when it is actually performing a "hard delete" of those before the `timedelta`.

```suggestion
        """Clean up (hard delete) all the soft deleted drafts.

        The drafts in the last timedelta span of time won't be deleted.
        """
```